src/autorest-core/resources/default-configuration.md

```diff
@@ -93,9 +93,33 @@ pipeline:
   swagger-document/semantic-validator:
     input: transform
     scope: semantic-validator
+```
+
+##### Azure Validator
+
+``` yaml
+pipeline:
   swagger-document/azure-validator:
     input: transform
-    scope: azure-validator
+    scope: azure-validator-composed
+  swagger-document/individual/azure-validator:
+    input: individual/transform
+    scope: azure-validator-individual
+```
+
+Activate `azure-validator` when setting `azure-arm`!?
+
+``` yaml $(azure-arm)
+azure-validator: true
+```
+
+``` yaml $(azure-validator)
+azure-validator-composed:
+  merge-state: composed
+  openapi-type: arm
```

Comment:
I would not state that in here (and below) due to duplication! even worse, since nested settings have more priority than "top level" ones, people cannot simply override those (which is kind of an important thing to enable)!

They would also have to write
```
azure-validator-composed:
  openapi-type: foo
azure-validator-individual:
  openapi-type: foo
```
which is horrible. Just set this default on the top level and everything is fine ;)